cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1595,166 +1595,280 @@ action.summary_index.alert_name = Splunk Admin - Missing Forwarder
+enableSched = 1
+alert.track = 0
+cron_schedule = 10 * * * *
+description = This report update the device inventory every hour and generates lookup for crowdstrike data. \
+\
+Data Collection - The crowdstrike data needs to be collected via the CrowdStrike Event Stream Add-on (https://splunkbase.splunk.com/app/5082/).
+dispatch.earliest_time = -62m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_crowdstrike_eventstream` | eval crowdstrike_userid=coalesce('event.UserId', 'event.UserName', user) | dedup crowdstrike_userid \
+| eval ip=lower(coalesce('event.UserIp', 'event.LocalIP', src)), hostname=lower(coalesce('event.ComputerName', 'event.HostnameField')) \
+| rename host as crowdstrike_collected_by, "metadata.customerIDString" as crowdstrike_customer_id, "event.MACAddress" as mac_address \
+| fillnull ip, hostname, mac_address value="" \
+| table _time, crowdstrike_collected_by, crowdstrike_customer_id, ip, hostname, mac_address, crowdstrike_userid \
+| append [| inputlookup cs_crowdstrike_inventory | rename time as _time] \
+| stats latest(_time) as time, latest(*) as * by crowdstrike_userid \
+| eval _key=crowdstrike_userid | outputlookup cs_crowdstrike_inventory \
+| `cs_device_inventory_out_to_temp_lookup`
+
+
+[Device Inventory Gen]
+disabled = 0
+enableSched = 1
+alert.track = 0
+cron_schedule = 20 * * * *
+description = This report update the device inventory every hour and generates device inventory lookup from device inventory temp lookup. \
+\
+Data Collection - This report depends on different sub-reports like: Device Inventory - Lansweeper, Tenable, Qualys, Sophos, Windows Defender, CrowdStrike.
+dispatch.earliest_time = -61m@m
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | inputlookup cs_device_inventory_temp \
+| addinfo | where time>=info_min_time and time<=info_max_time \
+| deviceinventorygen
+
+
+# Backfill
+[Device Inventory Backfill]
+disabled = 0
+enableSched = 0
+alert.track = 0
+run_on_startup = true
+description = This report backfills the all device inventory lookups. (Default timerange: last 30 days). \
+The report executes on Splunk start to make sure we do not miss data as well as this make sure user do not have make manually tasks.
+dispatch.earliest_time = -30d@m
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = | savedsearch "Device Inventory - Tenable Vuln" \
+| append [| search `cs_lansweeper` `cs_lansweeper_timerange` | `cs_lansweeper_inventory_fill_search`] \
```

Comment:
We don't need to backfill data for Lansweeper and Sophos.

Reply:
For backfill it will use same timerange now, so that should be fine.